src/definitions/modules/progress.js

```diff
@@ -394,19 +419,54 @@ $.fn.progress = function(parameters) {
         },
 
         set: {
-          barWidth: function(value) {
-            if(value > 100) {
-              module.error(error.tooHigh, value);
-            }
-            else if (value < 0) {
-              module.error(error.tooLow, value);
+          barWidth: function(values) {
+            module.debug("set bar width with ", values);
+            values = toArray(values);
+            var total = values.reduce(function(a,b) {
```

Comment:
Then central `sum` function could be used here (as mentioned above, this might be moved into the `module.helper` namespace)

Reply:
Done in https://github.com/fomantic/Fomantic-UI/pull/582/commits/1cf9677bb78449208231070fd6a25e122049fab9